akka-remote/src/main/resources/reference.conf

```diff
@@ -171,15 +167,40 @@ akka {
       log-sent-messages = off
 
       advanced {
+      
+        # Maximum serialized message size, including header data.  
+        maximum-frame-size = 256 KiB
+        
+        # Direct byte buffers are reused in a pool with this maximum size.
+        # Each buffer has the size of 'maximum-frame-size'.
+        buffer-pool-size = 128
+        
+        # Maximum serialized message size for the large messages, including header data.
+        # See 'large-message-destinations'.
+        maximum-large-frame-size = 2 MiB
+        
+        # Direct byte buffers for the large messages are reused in a pool with this maximum size.
+        # Each buffer has the size of 'maximum-large-frame-size'.
+        # See 'large-message-destinations'.
+        large-buffer-pool-size = 32
+      
         # For enabling testing features, such as blackhole in akka-remote-testkit.
         test-mode = off
 
         # Settings for the materializer that is used for the remote streams.
         materializer = ${akka.stream.materializer}
-        materializer {
-          dispatcher = "akka.remote.default-remote-dispatcher"
-        }
-
+        
+        # If set to a nonempty string artery will use the given dispatcher for
+        # the ordinary and large message streams, otherwise the default dispatcher is used.
+        use-dispatcher = "akka.remote.default-remote-dispatcher"
```

Comment:
it's a convention we have at other places, when it's a dispatcher name, i.e. a reference to a dispatcher configuration section (as opposed to the dispatcher section itself)